db/migrate/20210714144543_references_for_pull_request_comments.rb

```diff
@@ -0,0 +1,6 @@
+class ReferencesForPullRequestComments < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :pull_request_comments, :pull_request, null: false, foreign_key: true
+    add_reference :pull_request_comments, :owner, foreign_key: { to_table: :users }
```

Comment:
Maybe you can add `bulk` option to speed up the migration. WDYT?

https://blog.toshima.ru/2020/01/02/rails-migration-bulk-true.html

Reply:
Done! Thank you!